### PR TITLE
improving the docu for triggers

### PR DESCRIPTION
**Pull request by klinakuf**
This PR improves the docu for triggers. In addition to describing what triggers in spd are, it provides examples for common triggers.

### CODE REVIEW

File: spds/triggers.markdown [hunk 1]
```diff
@@ -7,13 +7,85 @@ permalink: /triggers/
...
+<hr>
+<img src="../images/simulation-time.png" alt="Simulation Time Trigger"/>
+<br>
+</details>
+
+### OperationResponseTime (SourceInterfaceStimulus)
+
+The OperationResponseTime is an example of a stimulus obtained from the SourceInterface of the target group. 
+This particular stimulus is based on the response time of an operation. For example, one can define a trigger that fires when the response time of an operation is 
+above 100ms.
```
Comment (stiesssh) [inline spds/triggers.markdown]: in your example below, it is 200ms.

File: spds/triggers.markdown [hunk 2]
```diff
@@ -7,13 +7,85 @@ permalink: /triggers/
...
+The SimpleFireOnValue allows to define elasticity policies of type threshold-based which are quite common in the cloud.
+
+The following figure shows the class diagram of the trigger hierarchy.
+
+![mermaid-triggers.png](../images/mermaid-triggers.png)
+
+## Common Stimulus and Expectations: Examples for SimpleFireOnValue
+
+The Stimulus defines observable information during the simulation of an architectural model upon which a trigger can fire. 
+There exist a variety of stimuli that can be used to define triggers. Please check the [Stimuli](../docu/#triggersStimulus) section for more information.
```
Comment (stiesssh) [inline spds/triggers.markdown]: not sure whether that link actually works as intended?

File: spds/triggers.markdown [hunk 3]
```diff
@@ -7,13 +7,85 @@ permalink: /triggers/
...
+  <summary>Example</summary>
+<br>
+An example with a SimpleFireOnValue trigger that fires when the response time of a particular operation (e.g., 'getItems') is above 200ms.
+<hr>
+<img src="../images/operation-response-time.png" alt="Operation Response Time Trigger"/>
+<br>
+</details>
+
+
+### CPUUtilization (ResourceUtilizationState --|> ManagedElementsState --|> TargetGroupState)
```
Comment (stiesssh) [inline spds/triggers.markdown]: * what does this "-|>" symbol mean? if it should be an arrow, i'd go for "->" which is imho more commonly used as an arrow. 
* what are these three states? where do they come from, have you already introduced them earier an, and i just forgot? o_O (writing them as a singles word in pascal case makes the look a bit like class names, which i fathom they are not?)

**Review by stiesssh — COMMENTED** [review]
lgtm, except for the points i mention in the comments.
i especially like the examples and also how you made them "foldable".

**Comment by klinakuf** [comment]
I could not regenerate the docu to fix the missing/broken links due to some errors which I cannot look into right now. I commented out the link to stimuli for now, docu has to be regenerated anyhow also for other parts and the links checked.

**Review by stiesssh — APPROVED** [review]
all previous concerns addressed, local deployment works just fine.

**Pull request merged**